homeassistant/components/publibike/config_flow.py

```diff
@@ -0,0 +1,83 @@
+"""Config flow for the PubliBike Public API integration."""
+
+from __future__ import annotations
+
+import logging
+
+from pypublibike.publibike import PubliBike
+from requests import RequestException
+import voluptuous as vol
+
+from homeassistant import config_entries
+from homeassistant.data_entry_flow import FlowResult
+from homeassistant.helpers import config_validation as cv
+
+from .const import (
+    BATTERY_LIMIT,
+    BATTERY_LIMIT_DEFAULT,
+    DOMAIN,
+    LATITUDE,
+    LONGITUDE,
+    STATION_ID,
+)
+
+_LOGGER = logging.getLogger(__name__)
+
+DATA_SCHEMA = vol.Schema(
+    {
+        vol.Required(BATTERY_LIMIT, default=BATTERY_LIMIT_DEFAULT): vol.All(
+            cv.positive_int, vol.Range(1, 100)
+        ),
```

Comment:
Isn't this an option? As in, is this really required to enter?

Reply:
Doesn't need to be - I made it optional.